cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -92,18 +84,58 @@ type certManagerClient struct {
 	configClient        config.Client
 	proxy               Proxy
 	pollImmediateWaiter PollImmediateWaiter
+	embeddedCertManagerManifestVersion string
+	embeddedCertManagerManifestHash string
 }
 
 // Ensure certManagerClient implements the CertManagerClient interface.
 var _ CertManagerClient = &certManagerClient{}
 
+func getManifestHash() string {
```

Comment:
this func seems to not be used down in the code, however I like the idea of hiding the logic for computing the hash into a func
```suggestion
func (cm *certManagerClient) getManifestHash() (string, err) {
```